spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'takeaway'
+
+describe Takeaway do
+  it "returns a menu when requested" do
+    expect(subject).to respond_to(:read_menu)
+  end
+
+  it "returns the full menu when requested" do
+    expect(subject.menu.dishes.length).to eq 5
```

Comment:
We could call and test the `read_menu` method here instead of mimicking the logic inside of that method:
```
expect(subject.read_menu).to eq(foo)
```

Reply:
test updated to:
expect(suject.read_menu).to include("pizza", "burger", "chow mein", "steak", "ice. cream")